dev-support/docker/Dockerfile_aarch64

```diff
@@ -33,61 +33,19 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV DEBCONF_TERSE true
 
 ######
-# Install common dependencies from packages. Versions here are either
-# sufficient or irrelevant.
+# Platform package dependency resolver
 ######
-# hadolint ignore=DL3008
+COPY pkg-resolver pkg-resolver
+RUN chmod a+x pkg-resolver/install-common-pkgs.sh pkg-resolver/resolve.py \
```

Comment:
We don't seem to be calling install-common-pkgs.sh?

Reply:
The hadolint in `Dockerfile_aarch64` is installed in a different manner compared to the rest of the platforms. Hence, I didn't invoke `install-common-pkgs.sh` here. But as per https://github.com/apache/hadoop/pull/3043#discussion_r641596725, I'll handle this as part of writing different `.sh` files for each library.

Reply:
We're using `pkg-resolver/resolve.py` in line 48.